Source/Utils/Source/Event/NativeEventDispatcher.cpp

```diff
@@ -23,21 +23,21 @@ namespace Internal
 NativeEventDispatcher::NativeEventDispatcher()
 	: EventDispatcher()
 	, EventHandleToContextMap()
-	, EventIdToHandleMap()
+	, EventIdToListenerHandlesMap()
 {
 }
 
 // -----------------------------------------------------------------------------
 
 void NativeEventDispatcher::FireEvent( const std::string& EventId, const Event& EventData )
 {
-	auto EventIdToHandleIterator = EventIdToHandleMap.find( EventId );
-	if ( EventIdToHandleIterator == EventIdToHandleMap.end() )
+	auto EventIdToListenerHandlesIterator = EventIdToListenerHandlesMap.find( EventId );
+	if ( EventIdToListenerHandlesIterator == EventIdToListenerHandlesMap.end() )
 	{
 		return;
 	}
 
-	for ( auto& EventHandle : EventIdToHandleIterator->second )
+	for ( auto& EventHandle : EventIdToListenerHandlesIterator->second )
```

Comment:
Not as important since I think it can be more easily inferred from context now, but do we want to rename the loop's `EventHandle` variable to `ListenerHandle` or `ListenerEventHandle` for consistency?

Reply:
The problem is then it's not consistent with `EventHandleToContextMap` which ultimately has a key of `EventType`. I think it's fine here.

Reply:
Er, the key for `EventHandleToContextMap` is of type `EventHandle` not `EventType`, right?  But yeah, I think it's more in line with the actual types this way.  I think the plural still makes it clearer, and since we have that I think it's fine. :)